Dockerfile.gravcore

```diff
@@ -1,4 +1,4 @@
-FROM bitnami/minideb:stretch
+FROM bitnami/minideb:latest
```

Comment:
I do not believe in referencing latest for OS baselines, due to not knowing changes between operating system versions, would like to keep this controlled.

Reply:
I understand, but with Buster works perfect

Reply:
More than happy to set it to buster.